yapapi/__init__.py

```diff
@@ -21,5 +23,14 @@ def get_version() -> str:
     return get_distribution("yapapi").version
 
 
+def asyncio_fix():
+    if sys.platform == "win32":
+
+        class _WindowsEventPolicy(asyncio.events.BaseDefaultEventLoopPolicy):
+            _loop_factory = asyncio.windows_events.ProactorEventLoop
+
+        asyncio.set_event_loop_policy(_WindowsEventPolicy())
```

Comment:
I'd just move those statements to the module level, outside any function

Reply:
then I'd need more "if sys.platform" etc since this class is not valid under non-Windows platforms ...

Reply:
ah, unless you mean the same as above - then, again, I'm unsure if this is a good idea to force that upon people ...